src/Contexts/Context.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect } from 'react';
+import { useLocalStorage } from '../hooks/useLocalStorage';
+
+export type DefaultValueType = {
+  todos: Todo[];
+  setTodos: (e: Todo[]) => void;
+  visibleTodos: Todo[];
+  setVisibleTodos: (e: Todo[]) => void;
+  query: string;
+  setQuery: (e: string) => void;
+  queryCondition: QueryConditions;
+  setQueryCondition: (e: QueryConditions) => void;
+};
+
+export type Todo = {
+  id: number;
+  title: string;
+  completed: boolean;
+};
+
+interface Props {
+  children: React.ReactNode;
+}
+
+export type QueryConditions = 'all' | 'active' | 'completed';
+
+export const todoContext = React.createContext<DefaultValueType | null>(null);
+
+export const AppContext: React.FC<Props> = ({ children }) => {
+  const [todos, setTodos] = useLocalStorage<Todo[]>('todos', [
+    // { id: +new Date(), completed: false, title: 'fiska' },
```

Comment:
Remove comment. Try it write obvious code that explain all we need without comments) Fix all cases